workloads/hyper-scale/common.sh

```diff
@@ -19,14 +20,16 @@ prep(){
         popd
     fi
     if [[ -z $(rosa version)  ]]; then
-        sudo curl -L $(curl -s https://api.github.com/repos/openshift/rosa/releases/latest | jq -r ".assets[] | select(.name == \"rosa-linux-amd64\") | .browser_download_url") --output /usr/local/bin/rosa
-        sudo curl -L $(curl -s https://api.github.com/repos/openshift-online/ocm-cli/releases/latest | jq -r ".assets[] | select(.name == \"ocm-linux-amd64\") | .browser_download_url") --output /usr/local/bin/ocm
-        sudo chmod +x /usr/local/bin/rosa && chmod +x /usr/local/bin/ocm
+        mkdir -p ${TEMP_DIR}/bin/
+        sudo curl -L $(curl -s https://api.github.com/repos/openshift/rosa/releases/latest | jq -r ".assets[] | select(.name == \"rosa-linux-amd64\") | .browser_download_url") --output ${TEMP_DIR}/bin/rosa
+        sudo curl -L $(curl -s https://api.github.com/repos/openshift-online/ocm-cli/releases/latest | jq -r ".assets[] | select(.name == \"ocm-linux-amd64\") | .browser_download_url") --output ${TEMP_DIR}/bin/ocm
+        sudo chmod +x ${TEMP_DIR}/bin/rosa && chmod +x ${TEMP_DIR}/bin/ocm
+        export PATH=${TEMP_DIR}/bin/:$PATH
```

Comment:
Training / again. Just for consistency w/ what is normally in $PATH

Reply:
Done, updated!